website/src/app/[lang]/[region]/(website)/campaign/[campaign]/page.tsx

```diff
@@ -47,127 +65,177 @@ export default async function Page({ params }: CampaignPageProps) {
 
 	return (
 		<>
-			<BaseContainer className="-mx-2 flex flex-wrap">
-				<div className="mb-4 w-full px-2 lg:w-1/2">
-					<Typography size="xl" weight="bold" color="accent">
-						{translator.t('campaign.by', { context: { creator: campaign.creator_name } })}
-					</Typography>
-					<Typography size="5xl" weight="bold" color="accent" className="mt-2">
-						{campaign.title}
-					</Typography>
-				</div>
-			</BaseContainer>
-			<BaseContainer className="-mx-2 flex flex-wrap">
-				<div className="mb-4 w-full px-2 lg:w-1/2">
-					<Typography size="lg" weight="medium">
-						{campaign.description}
-					</Typography>
-				</div>
-				<div className="mb-4 w-full px-2  lg:w-1/2">
-					<div>
-						{!campaign.goal && (
```

Comment:
The idea of this part is that we support campaigns without any goal and therefore without any progress bar. I think that's quite nice for some events where the creator doesn't want to set a specific goal.

Do you think you could readd this part? You can test it if you set the goal to 0 in the admin.